tests/Metalama.Framework.Tests.Integration/Tests/Aspects/DesignTimeInvalidCode/ErrorTypedConstantInAttribute.cs

```diff
@@ -0,0 +1,53 @@
+using System;
```

Comment:
I'm not sure this tests something else than the other test. Was this file supposed to include `// @TestScenario(DesignTime)`?